src/components/Hero/Hero.tsx

```diff
@@ -1,5 +1,12 @@
 import type { JSX } from 'react'
 
+import { Chip } from '../Chip'
+
 export function Hero(): JSX.Element {
-  return <h1 className="main-heading">Modern fullstack framework</h1>
+  return (
+    <>
+      <Chip>just published v0.19.4</Chip>
```

Comment:
We should probably have capital letter here
```suggestion
      <Chip>Just published v0.19.4</Chip>
```